lib/tests/e2e/suite.go

```diff
@@ -30,7 +30,6 @@ func NewE2ETestSuite(cfg network.Config) *E2ETestSuite {
 // SetupSuite initializes machine E2ETestSuite
 func (s *E2ETestSuite) SetupSuite() {
 	s.T().Log("setting up e2e test suite")
-
```

Comment:
this diff is unnecessary. please put back the newline to keep the diff as small as possible.